doc/source/theory/multiphase/cfd/vof.rst

```diff
@@ -111,7 +111,7 @@ Interface Diffusion and Sharpening
 
 The VOF method tends to diffuse the interface, i.e., over time, the interface becomes blurry instead of a sharp definition, and the change from :math:`\phi = 0` to :math:`1` happens on a larger distance.
```

Comment:
```suggestion
The VOF method tends to diffuse the interface, i.e., over time, the interface becomes blurry instead of a sharp definition, and the change from :math:`\phi = 0` to :math:`1` occurs on a larger distance.
```